src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -2749,7 +2749,7 @@ internal BuildingRequestEventArgs CreateRequestArgsAndFireBuildingRequest(string
         /// <param name="languageDependentNamespace">Namespace of the type, can be different in VB than the fullNamespace.</param>
         /// <returns>Type that the name resolved to or null if none found.</returns>
         /// <remarks>Function was added for Portable Lib support to handle the differences in accessing the assembly of the context.</remarks>
-        protected internal Type DefaultResolveType(string typeName, string fullNamespace, string languageDependentNamespace)

```

Comment:
>protected internal [](start = 8, length = 18)

I don't think to change this "internal" as public? What's the reason why we should do like this.